website/content/docs/getting-started/dev-mode/connect-to-dev-target.mdx

```diff
@@ -1,8 +1,8 @@
 ---
 layout: docs
 page_title: Connect to your first target
-description: |-
-  Connecting to your first target
+description: >-
+  Use Boundary's dev mode to learn how to connect to targets and how to optionally use connect helpers, exec flag, SSH ProxyCommand, and Desktop Client.
```

Comment:
You can omit saying "dev mode" since what's explained on this page is relevant with non-dev environments as well.

```suggestion
  Learn how to connect to targets using Boundary, and optionally use connect helpers, exec flag, SSH ProxyCommand, and Desktop Client features.
```